src/Sarif.Multitool.Library/Rules/Base1013.ProvideRequiredSarifLogProperties.cs

```diff
@@ -12,7 +12,15 @@ public class BaseProvideRequiredSarifLogProperties
 
         public override HashSet<RuleKind> RuleKinds => new HashSet<RuleKind>();
 
-        public override MultiformatMessageString FullDescription => new MultiformatMessageString { Text = RuleResources.SARIF2008_ProvideSchema_FullDescription_Text };
+        public override MultiformatMessageString FullDescription => new MultiformatMessageString { Text = RuleResources.Base1013_ReferenceFinalSchema_FullDescription_Text };
```

Comment:
>Base1013_ReferenceFinalSchema_FullDescription_Text [](http://example.com/codeflow?start=120&length=50)

Is this fix correct? Or was the old value by design?

Reply:
Either is fine. I don't provide a value for a lot of these so we will have to fix that at some point.

Reply:
got it. Kept my change then, 
because consistent name will confuse less people.